scripts/pull_github_pr.sh

```diff
@@ -115,12 +115,23 @@ PR_TITLE=$(jq -r .title <<< $PR_DATA)
 echo "    $PR_TITLE"
 PR_DESCR=$(jq -r .body <<< $PR_DATA)
 PR_LOGIN=$(jq -r .head.user.login <<< $PR_DATA)
+PR_LABELS=$(jq -r .labels[].name <<< "$PR_DATA")
+PR_BODY=$(jq -r .body <<< "$PR_DATA")
 echo -n "Fetching full name of author $PR_LOGIN... "
 USER_NAME=$(curl -s "https://api.github.com/users/$PR_LOGIN" | jq -r .name)
 echo "$USER_NAME"
 
 git fetch "$REMOTE" pull/$PR_NUM/head
 
+if [[ "$FORCE" != "--skip-validation" ]]; then
```

Comment:
This means we cannot both skip the jenkins check *and* the validation.
I think it is worth adding a separate flag for this (and also making these flags truly non-positional).

Reply:
will fix it